source-browser.html

```diff
@@ -92,56 +152,45 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
             });
           });
         }
-
         if (cacheStale) {
           console.warn('Repository list cache stale, updating.');
-
           const response = await _fetchRepos(1);
           _addRepos(await response.json());
           const linkHeader = response.headers.get('Link');
           const links = linkHeader ? linkHeader.split(',').map(x => x.split('; ')) : [];
           const lastPage = links.length ? parseInt(links.find(([_, rel]) => rel === 'rel="last"')[0].match(/page=(\d+)/)[1]) : 1;
-
           const remainingPages = [...Array(lastPage + 1).keys()].slice(2);
           const responses = await Promise.all(remainingPages.map(_fetchRepos));
           const reposs = await Promise.all(responses.map(response => response.json()));
           reposs.forEach(_addRepos);
-
           localStorage[CACHE_KEY] = JSON.stringify(reposByTopic);
           localStorage[EXPIRY_KEY] = Date.now() + EXPIRY_MILLIS;
         } else {
           reposByTopic = JSON.parse(localStorage[CACHE_KEY]);
         }
-
         return Promise.resolve(reposByTopic);
       }
-
       async function showTopics() {
         $('#topics, #loading').toggleClass('display-none');
-
         const reposByTopic = await getReposByTopic(ORGANIZATION);
         const normalizedSearchText = $('#search')
           .val()
           .toLowerCase();
-
         $('.topic').remove();
-
         TOPICS.forEach(({ topic, name, repoUrl }, i) => {
-          const repoList = $('<ul class="repos list-reset mt0">');
-
+          const repoList = $('<ul id="'+ name +'" class="repos list-reset mt0">');
```

Comment:
Please use jQuery's built-in attribute object parameter.

Reply:
Okay!

Reply:
That change isn't necessary.. So Done ;)